src/main/java/org/taktik/icure/services/external/rest/v1/facade/ContactFacade.java

```diff
@@ -270,6 +270,30 @@ public Response findByHCPartyFormIds(@QueryParam("hcPartyId") String hcPartyId,
 		}
 	}
 
+    @ApiOperation(
+            value = "List contacts found By Healthcare Party and Patient foreign keys.",
+            response = ContactDto.class,
+            responseContainer = "Array",
+            httpMethod = "POST"
+    )
+    @POST
+    @Path("/byHcPartyPatientForeignKeys")
+    public Response findByHCPartyPatientForeignKeys(@QueryParam("hcPartyId") String hcPartyId, ListOfIdsDto patientForeignKeys) {
+        if (hcPartyId == null || patientForeignKeys == null || patientForeignKeys.getIds().size()==0) {
```

Comment:
`patientForeignKeys.getIds().size()==0`
we dont have file formatting watchers on backend ? :)